mod/developers/start.php

```diff
@@ -14,8 +14,8 @@ function developers_init() {
 
 	elgg_register_plugin_hook_handler('register', 'menu:page', '_developers_page_menu');
 		
-	elgg_extend_view('admin.css', 'developers/css');
-	elgg_extend_view('elgg.css', 'developers/css');
+	elgg_extend_view('admin.css', 'developers.scss');
```

Comment:
@Elgg/core Are we cool with using a `*.scss`? Here it Just Works because it's thrown into CSS views, but are we better off leaving invalid CSS syntax in `.css` files?

Reply:
I'd rather we use `.crush` or something and tell IDE to use scss syntax, otherwise it gets messy.